packages/lesswrong/components/quickTakes/QuickTakesSection.tsx

```diff
@@ -15,25 +32,60 @@ const QuickTakesSection = ({classes}: {
   classes: ClassesType,
 }) => {
   const currentUser = useCurrentUser();
+
   const {expanded, toggleExpanded} = useExpandedFrontpageSection({
     section: "quickTakes",
     defaultExpanded: "all",
     onExpandEvent: "quickTakesSectionExpanded",
     onCollapseEvent: "quickTakesSectionCollapsed",
     cookieName: SHOW_QUICK_TAKES_SECTION_COOKIE,
   });
-  const {ExpandableSection, QuickTakesEntry, QuickTakesList} = Components;
+
+  const {
+    expanded: showCommunity,
+    toggleExpanded: toggleShowCommunity,
+  } = useExpandedFrontpageSection({
+    section: "quickTakesCommunity",
+    defaultExpanded: (currentUser: UsersCurrent | null) =>
+      currentUser?.hideCommunitySection
+        ? false
+        : !!currentUser?.expandedFrontpageSections?.community,
```

Comment:
I think this interaction is a bit weird. The "Show community" quick takes checkbox depends on the previous time I expanded/collapsed the "Community posts" section, until I explicitly check or uncheck the "Show community" checkbox. So like, if I expand "Community posts" now, upon refresh, the checkbox is checked.

Maybe the thing we want is to set the cookie the first time the user loads this section. That way it won't sometimes change on them unexpectedly.